src/assert.js

```diff
@@ -65,16 +74,31 @@ function assertion(errorCls, shouldBeTruthy, opt_message, var_args) {
     return shouldBeTruthy;
   }
 
+  // Skip the first 3 arguments to isolate format params
+  const messageArgs = Array.prototype.slice.call(arguments, 3);
+  const messageArray = [];
+  let firstElement;
+
   // Substitute provided values into format string in message
-  const message = Array.prototype.slice
-    // Skip the first 3 arguments to isolate format params
-    .call(arguments, 3)
-    .reduce(
-      (msg, subValue) => msg.replace('%s', subValue),
-      opt_message || 'Assertion failed'
-    );
+  const message = (opt_message || 'Assertion failed').replace(/%s/g, () => {
+    const subValue = messageArgs.shift();
+
+    if (subValue != '') {
+      messageArray.push(subValue);
+    }
+
+    // If an element is provided, add it to the error object
+    if (!firstElement && subValue?.nodeType == 1) {
+      firstElement = subValue;
+    }
+
+    return subValue;
+  });
```

Comment:
@rsimha Do you know why this might thing these lines were not covered by tests? Every line is indeed covered

Reply:
Not sure why it claimed zero coverage.

From CI logs, unit test coverage was successfully uploaded for the last commit in this PR: https://app.circleci.com/pipelines/github/ampproject/amphtml/2836/workflows/8eda1669-cac3-4370-a4f3-548652ffaf9f/jobs/32913/parallel-runs/0/steps/0-110

And codecov shows that the lines added by this PR are indeed covered (from running tests after this landed on `master`): https://codecov.io/gh/ampproject/amphtml/commit/ee8ac1edf18b76d00158ef949587b7e272fdad8a/ (link needs GH login)

Perhaps there was a blip the during PR build's coverage processing? Not sure. Can keep an eye, but can't think of any action items now.

Reply:
Unless... you changed the code in one commit and added tests in a follow up commit, and these statuses are from the first commit that didn't have the coverage?